### ISSUE TITLE
WCAG 1.4.1: Ensure links are distinguished from surrounding text in a way that does not rely on color (.prc-Link-Link-85e08)

### ISSUE THREAD

**Issue by Jinssi** [Jinssi/Keilamenu#2]
#### Issue
`Ensure links are distinguished from surrounding text in a way that does not rely on color` ([link-in-text-block](https://dequeuniversity.com/rules/axe/4.9/link-in-text-block?application=msftAI))
#### Target application
[Jinssi/chaos-engineering-demo: This repo holds the demo application used for the Health Modeling & Chaos Engineering engagement from Microsoft Customer Success](https://github.com/Jinssi/chaos-engineering-demo)
#### Element path
.crArEc > span > .prc-Link-Link-85e08
#### Snippet
`<a class="prc-Link-Link-85e08" href="/Jinssi/chaos-engineering-demo/compare/Azure%3Achaos-engineering-demo%3Amain...main">1 commit ahead of</a>`
#### How to fix
    Fix any of the following:
      The link has insufficient color contrast of 2.84:1 with the surrounding text. (Minimum contrast is 3:1, link text: #4493f8, surrounding text: #f0f6fc)
      The link has no styling (such as underline) to distinguish it from the surrounding text
#### Environment
Microsoft Edge version 132.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
